src/auth-service/controllers/create-user.js

```diff
@@ -286,6 +286,58 @@ const createUser = {
       });
     }
   },
+  deleteMobileUserData: async (req, res) => {
+    try {
+      let responseFromDeleteAppData;
+      logText("We are deleting the app data.....");
+      const { uid } = req.query;
+      const { ct } = req.query;
+
+      let request = {};
+      request["body"] = {};
+      request["body"]["uid"] = uid;
+      request["body"]["ct"] = ct;
+      logObject("request:", request);
+     try {
+       responseFromDeleteAppData = await createUserUtil.deleteMobileUserData(
+         request
+       );
+     } catch (error) {
+       console.error(error);
+     }
+      
+      logObject("responseFromDeleteAppData", responseFromDeleteAppData);
+
+      if (responseFromDeleteAppData.success === true) {
+        const status = responseFromDeleteAppData.status
+          ? responseFromDeleteAppData.status
+          : httpStatus.OK;
+        res.status(status).json({
+          success: true,
+          message: "Data deleted sucessfully",
+        });
+      } else if (responseFromDeleteAppData.success === false) {
+        const status = responseFromDeleteAppData.status
+          ? responseFromDeleteAppData.status
+          : httpStatus.INTERNAL_SERVER_ERROR;
+        return res.status(status).json({
+          success: false,
+          message: responseFromDeleteAppData.message,
+          errors: responseFromDeleteAppData.errors
+            ? responseFromDeleteAppData.errors
+            : { message: "internal server errors" },
+        });
+      }
+
+    } catch (error) {
+      return {
+        success: false,
+        message: "Internal Server Error",
+        status: httpStatus.INTERNAL_SERVER_ERROR,
```

Comment:
add an errors object, more like: 
```
{
message: "Internal Server Error",
status: httpStatus.INTERNAL_SERVER_ERROR,
errors:{message:error.message}
}
```

And please remember that this is a controller file so the return statement should start like `res.status(status).json({RESPONSE_OBJECT_ABOVE})`

Reply:
Thanks for catching this @Baalmart